src/http2_stream.erl

```diff
@@ -546,3 +564,21 @@ rst_stream_(ErrorCode,
                       RstStream}),
     sock:send(Socket, RstStreamBin),
     ok.
+
+check_content_length(Stream) ->
+    ContentLength =
+        proplists:get_value(<<"content-length">>,
+                            Stream#stream_state.request_headers),
+
+    case ContentLength of
```

Comment:
A response
   that is defined to have no payload, as described in [RFC7230],
   Section 3.3.2, can have a non-zero content-length header field, even
   though no content is included in DATA frames. 
https://tools.ietf.org/html/rfc7540#section-8.1.2.6